### PR TITLE
dm: enhance description about error triggering condition

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Enhance description about error triggering condition

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/12301
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by D3Hunter — COMMENTED** [review]
lgtm. cc @lance6716

**Review by D3Hunter — APPROVED** [review]

File: dm/dm-handle-alerts.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ This document introduces how to deal with the alert information in DM.
 
 - Description:
 
-    When the relay log processing unit encounters an error, this unit moves to `Paused` state, and an alert is triggered immediately.
+    When the relay log processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this unit moves to `Paused` state, and an alert is triggered immediately.
```
Comment (hfxsd) [inline dm/dm-handle-alerts.md:77]:
```suggestion
    When the relay log processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this alert is triggered immediately.
```

File: dm/dm-handle-alerts.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ This document introduces how to deal with the alert information in DM.
 
 - Description:
 
-    When the Dump processing unit encounters an error, this unit moves to the `Paused` state, and an alert is triggered immediately.
+    When the Dump processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this unit moves to the `Paused` state, and an alert is triggered immediately.
```
Comment (hfxsd) [inline dm/dm-handle-alerts.md:143]:
```suggestion
    When the Dump processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this alert is triggered immediately.
```

File: dm/dm-handle-alerts.md [hunk 3]
```diff
@@ -150,7 +150,7 @@ This document introduces how to deal with the alert information in DM.
 
 - Description:
 
-    When the Load processing unit encounters an error, this unit moves to the `Paused` state, and an alert is triggered immediately.
+    When the Load processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this unit moves to the `Paused` state, and an alert is triggered immediately.
```
Comment (hfxsd) [inline dm/dm-handle-alerts.md:153]:
```suggestion
    When the Load processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this alert is triggered immediately.
```

File: dm/dm-handle-alerts.md [hunk 4]
```diff
@@ -162,7 +162,7 @@ This document introduces how to deal with the alert information in DM.
 
 - Description:
 
-    When the binlog replication processing unit encounters an error, this unit moves to the `Paused` state, and an alert is triggered immediately.
+    When the binlog replication processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this unit moves to the `Paused` state, and an alert is triggered immediately.
```
Comment (hfxsd) [inline dm/dm-handle-alerts.md:165]:
```suggestion
    When the binlog replication processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors in a short period of time (for example, network problems), this alert is triggered immediately.
```

**Review by lance6716 — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: dm/dm-handle-alerts.md [hunk 5]
```diff
@@ -74,7 +74,7 @@ This document introduces how to deal with the alert information in DM.
 
 - Description:
 
-    When the relay log processing unit encounters an error, this unit moves to `Paused` state, and an alert is triggered immediately.
+    When the relay log processing unit encounters a non-autorecoverable error (for example, binlog file not found), or when it encounters multiple recoverable errors (for example, network problems) in a short period of time (for example, more than 3 times in 2 minutes), this alert is triggered immediately.
```
Comment (qiancai) [inline dm/dm-handle-alerts.md:77]:
```suggestion
    When the relay log processing unit encounters a non-autorecoverable error (for example, binlog files not found), or when it encounters multiple recoverable errors (for example, network problems) in a short period of time (for example, more than 3 times in 2 minutes), this alert is triggered.
```

Comment (qiancai) [reply dm/dm-handle-alerts.md:77]: Please also update other places for this sentence.

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**